library/src/webapp/editor/ckextraplugins/creative-commons-images/js/bind-creative-commons-image-search-to-dialog.js

```diff
@@ -40,7 +40,7 @@ var BindCreativeCommonsImageSearchToDialog = function(path, dialog) {
   // now initialize iframe which will isolate the search field (so we can submit
   // the form without closing the dialog)
   var pushFormIntoIframe = function() {
-    var iframe = $('<iframe src="about:blank"></iframe>').attr('id', 'creativeCommonsImageSearchIframe');
+    var iframe = $('<iframe src="about:blank"></iframe>').attr('id', 'creativeCommonsImageSearchIframe').attr('style', 'height: 50px;width: 100%;');
```

Comment:
You don't need to fix this, but for multiple options you can pass in a object  to the attr call and set them all in one go. Also for CSS changes you can use the css function.